guides/source/generators.md

```diff
@@ -21,18 +21,21 @@ After reading this guide, you will know:
 First Contact
 -------------
 
-When you create an application using the `rails` command, you are in fact using a Rails generator. After that, you can get a list of all available generators by just invoking `rails generate`:
+When you create an application using the `rails` command, you are in fact using a Rails generator. After that, you can get a list of all available generators by just invoking `bin/rails generate`:
 
 ```bash
 $ rails new myapp
 $ cd myapp
-$ rails generate
+$ bundle
```

Comment:
You should not need to run `bundle` here.

Reply:
not sure how that got there, will fix...